src/flights/url.py

```diff
@@ -0,0 +1,48 @@
+from fast_flights import FlightData, Passengers, create_filter
+
+def generate_flight_url(
+        departure_date,
+        from_airport,
+        to_airport,
+        trip_type="one-way",
+        seat_type="economy",
+        adults=1,
+        children=0,
+        infants_in_seat=0,
+        infants_on_lap=0,
+        currency="EUR"
```

Comment:
@liliandoublet let's type the code chef
ex: `departure_date: str`, `adults: int=1`, `currency: str='EUR'`..

add docstrings:
'''
Generates flight url from parameters:
- departure date: str, YYYY-mm-dd formatted.
- ...
'''